src/java/edu/slu/tpen/entity/image/Canvas.java

```diff
@@ -254,63 +255,63 @@ public static JSONArray getLinesForProject(Integer projectID, String canvasID, I
      * look like an otherContent field.
      */
     public static JSONArray getAnnotationLinesForAnnotationPage(Integer projectID, String canvasID,Integer folioNumber, Integer UID, String profile) throws MalformedURLException, IOException, SQLException {
-        //System.out.println("Get lines for project");
         JSONObject annotationPage = new JSONObject();
-        JSONArray resources_array = new JSONArray();
+        JSONArray annotationsArray = new JSONArray();
         Annotation[] annotations = null;
 //        ArrayList<Annotation> annotations = new ArrayList<Annotation>();
         String dateString = "";
         String annoListID = getRbTok("SERVERURL") + "project/" + projectID + "/annotations/" + folioNumber;
-        annotationPage.element("id", annoListID);
-        annotationPage.element("type", "AnnotationPage");
-//        annotationPage.element("label",buildLanguageMapOtherContent("en",canvasID));
-        //annotationList.element("proj", projectID);
-        annotationPage.element("target", canvasID);
-        if (profile.contains("v3")){
-            System.out.println("reached v3"); //System.out.println("Put all canvas together");
-           List<Map<String, Object>> pageList = new ArrayList<>();
-            Project proj = new Project(projectID);
-            String projName = proj.getName();
-            User u = new User(UID);
-            Folio[] folios = proj.getFolios();
+//        annotationPage.element("id", annoListID);
```

Comment:
When chunks like this really aren't needed, they can be removed. Versioning will leave them available to us if we need to rewind time.